src/h5web/providers/mock/data.ts

```diff
@@ -271,6 +272,18 @@ export const mockMetadata = makeMetadata({
         makeGroupHardLink('log_spectrum'),
       ]
     ),
+    makeGroup('nexus_malformed', undefined, [
+      makeGroupHardLink('default_not_string'),
+      makeGroupHardLink('default_empty'),
+      makeGroupHardLink('default_not_found'),
+      makeGroupHardLink('no_signal'),
+      makeGroupHardLink('signal_not_string'),
+      makeGroupHardLink('signal_not_found'),
+      makeGroupHardLink('signal_not_dataset'),
+      makeGroupHardLink('signal_dataset_not_numeric'),
+      makeGroupHardLink('signal_dataset_not_simple'),
+      makeGroupHardLink('signal_dataset_zero_dim'),
```

Comment:
Every error thrown by `supportsEntity` is covered. This will allow me to replace some of the `supportsEntity` unit tests with feature tests (with `renderApp()`). I'll do this in another PR.

Reply:
Done in this PR after all.